src/core/agent/services/credentialService.ts

```diff
@@ -402,14 +402,18 @@ class CredentialService extends AgentService {
     });
   }
 
-  async getUnhandledCredentials(): Promise<
-    (CredentialExchangeRecord | KeriNotification)[]
-    > {
+  async getUnhandledCredentials(filters?: {
+    isDismissed?: boolean;
+  }): Promise<(CredentialExchangeRecord | KeriNotification)[]> {
+    let otherFilters = {};
+    if (typeof filters?.isDismissed === "boolean") {
+      otherFilters = { isDismissed: filters?.isDismissed };
+    }
```

Comment:
Hmm... can we just have `filters: { isDismissed?: boolean } = {}` and just pass this directly to `getKeriCredentialNotifications`, and in `getKeriCredentialNotifications` just pass this directly to `findAllByQuery`.

This would be easier than doing this conversion thing here.